config/initializers/session_store.rb

```diff
@@ -1,9 +1,16 @@
 service_url = ENV['SERVICE_URL']
 SERVICE_DOMAIN = (service_url) ? URI.parse(service_url).host : nil
+timeout_time = ENV['TIMEOUT_TIME']
+
+if Rails.env.developement?
```

Comment:
We shouldn't be switching on `Rails.env` values; this should entirely be handled by the environment variables, possibly with a default.